docs/_getting-started/index.md

```diff
@@ -4,197 +4,311 @@ permalink: docs/getting-started
 sort_key: A
 ---
 
-To use Geth, you need to install it first. You can install Geth in a variety
-of ways that you can find in the "[Install and Build](install-and-build/installing-geth)" section.
-These include installing it via your favorite package manager, downloading a
-standalone pre-built binary, running it as a docker container or building it yourself.
+To use Geth, you need to install it first. You can install Geth in a variety of ways that you can find in the “[Install and Build](install-and-build/installing-geth)” section. 
+These include installing it via your favorite package manager, downloading a standalone pre-built binary, running it as a docker container, or building it yourself.
 
-For this guide, we assume you have Geth installed and are ready to find out how to use it.
-The guide shows you how to create accounts, sync to a network, and then send transactions
-between accounts.
+For this guide, we assume you have Geth installed and are ready to find out how to use it. 
+The guide shows you how to create accounts, sync to a network, and then send transactions between accounts.
+This guide uses [Clef](clef/tutorial), which is our preferred tool for signing transactions with Geth, and will replace Geth’s account management.
 
-This guide uses [Clef](clef/tutorial), which is our preferred tool for signing transactions with Geth,
-and will replace Geth's account management.
+### Two Important terms in geth:
```

Comment:
I didn't notice this until checking in Jekyll. I think this listing is a bit out of place here.